pkg/master/tunneler/ssh.go

```diff
@@ -83,7 +83,7 @@ type SSHTunneler struct {
 	InstallSSHKey  InstallSSHKey
 	HealthCheckURL *url.URL
 
-	tunnels        *ssh.SSHTunnelList
+	tunnels        *ssh.TheTunnelList
```

Comment:
i am curious to see what golint errors required renaming this ?

Reply:
It is programmed to prevent "redundancy" in naming. The problem is actually in the pkg/ssh/ssh.go file. It wants no public variables in a package foo (or ssh) to begin with foo (or ssh).